src/App.js

```diff
@@ -1,5 +1,100 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+
+const RandomBallNumber = () => {
+  const COMPUTER = [];
+  while (COMPUTER.length < 3) {
+    const NUMBER = Random.pickNumberInRange(1, 9);
+    if (!COMPUTER.includes(NUMBER)) {
+      COMPUTER.push(NUMBER);
+    }
+  }
+  return COMPUTER.join('');
+}
+
+const WRONG_NUMBER = (my_result) => {
+  if (my_result.length !== 3) {
+    throw new Error('\x1b[31m[ERROR] 3개만 입력해주세요\x1b[37m');
+  }
+
+  if (isNaN(my_result[0]) === true || isNaN(my_result[1]) === true || isNaN(my_result[2]) === true) {
+    throw new Error('\x1b[31m[ERROR] 숫자만 입력해주세요\x1b[37m');
+  }
+
+  if (my_result[0] === my_result[1] || my_result[0] === my_result[2] || my_result[1] === my_result[2]) {
+    throw new Error('\x1b[31m[ERROR] 서로 다른 숫자를 입력해주세요\x1b[37m');
+  }
+  if (my_result[0] == 0 || my_result[1] == 0 || my_result[2] == 0) {
+    throw new Error('\x1b[31m[ERROR] 1 ~ 9까지의 숫자를 입력해주세요\x1b[37m');
+  }
+}
+
+const WRONG_EXIT_CODE = (exit_code) => {
+  if (exit_code.length != 1) {
+    throw new Error('\x1b[31m[ERROR] 1개만 입력해주세요\x1b[37m');
+  }
+  if (isNaN(exit_code[0]) === true) {
+    throw new Error('\x1b[31m[ERROR] 숫자만 입력해주세요\x1b[37m');
+  }
+  if (exit_code[0] != 1 && exit_code[0] != 2) {
+    throw new Error('\x1b[31m[ERROR] 1과 2 둘 중 하나만 입력해주세요\x1b[37m');
+  }
+}
+
+const print_cnt = (BallCnt, StrikeCnt) => {
+  if (BallCnt > 0 && StrikeCnt > 0) {
+    Console.print(`${BallCnt}볼 ${StrikeCnt}스트라이크`);
+  }
+  if (BallCnt > 0) {
+    Console.print(`${BallCnt}볼`);
+  }
+  if (StrikeCnt > 0) {
+    Console.print(`${StrikeCnt}스트라이크`);
+  }
+  if (BallCnt === 0 && StrikeCnt === 0) {
+    Console.print(`낫싱`);
+  }
+}
+
 class App {
-  async play() {}
+  async play() {
+
+    Console.print('\x1b[37m숫자 야구 게임을 시작합니다.');
+    while (1) {
+      const COMPUTER_NUMBER = RandomBallNumber();
+      while (1) {
+        const my_result = await Console.readLineAsync('숫자를 입력해주세요 : ');
+
+        WRONG_NUMBER(my_result);
+
+        let BallCnt = 0;
+        let StrikeCnt = 0;
```

Comment:
변수명은 camelCase로 작성하시는게 좋아보입니다!
그리고 저는 개인적으로 변수명을 줄이지않고 풀네임으로 작성하는걸 선호합니다

https://github.com/airbnb/javascript#naming--camelCase

```suggestion
        let ballCount = 0;
        let strikeCount = 0;
```

Reply:
count를 항상 cnt로 사용했었는데 줄이지 안혹 사용해야겠어요!